app/views/posts/index.html.erb

```diff
@@ -1 +1,15 @@
-<h1>Here is a list of all the posts</h1>
\ No newline at end of file
+<img class="pic" src="<%= @user.photo %>" alt="profile_pic">
+<%=@user.name %>
+<span>number of posts:  <%= @user.posts_counter%> </span>
+
+<% @posts.each do |post| %>
+
+<p><%= link_to post.title, user_post_path(@user,post)%></p>
+<p><%= post.text%></p>
+<p>comment:<%=post.comments_counter %></p>
+<p>likes:<%=post.likes_counter%></p>
+<h3> comments </h3>
+ <%post.comments.each do |comment|%>
+ <p><%=comment.author.name %>: <%=comment.text %></p>
+ <%end%>
```

Comment:
- [ ] You should pay more attention to the function as styling is nice to have the **See all Post** links open the user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png) However your design does not match the wireframe👇🏻 Kindly make sure that the index view page matches the given  [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png).

Your Implementation | Original Design
:---:|:---:
![image](https://user-images.githubusercontent.com/79058364/233747837-b214370e-0ee1-47bc-8144-56299fa1e09b.png) | ![image](https://user-images.githubusercontent.com/79058364/231671531-0b0de30c-f840-4c97-a525-15ae6cc6da35.png)
 
- Do the same for your Post show page.

![image](https://user-images.githubusercontent.com/79058364/233747955-1da783cd-6000-4350-ac6c-767ce938fceb.png)